x-pack/packages/kbn-elastic-assistant/impl/knowledge_base/knowledge_base_settings_management.tsx

```diff
@@ -0,0 +1,379 @@
+    (enabled: boolean) => {
+      if (enabled) {
+        setupKB();
+      } else {
+        deleteKB();
+      }
+    },
+    [deleteKB, setupKB]
+  );
+
+  const knowledgeBaseActionButton = useMemo(() => {
+    return isLoadingKb || !isKnowledgeBaseAvailable ? (
+      <></>
+    ) : (
+      <EuiButtonEmpty
+        color={isKnowledgeBaseEnabled ? 'danger' : 'primary'}
+        flush="left"
+        data-test-subj={'knowledgeBaseActionButton'}
+        onClick={() => onEnableKB(!isKnowledgeBaseEnabled)}
+        size="xs"
+      >
+        {isKnowledgeBaseEnabled
+          ? i18n.KNOWLEDGE_BASE_DELETE_BUTTON
+          : i18n.KNOWLEDGE_BASE_INIT_BUTTON}
+      </EuiButtonEmpty>
+    );
+  }, [isKnowledgeBaseAvailable, isKnowledgeBaseEnabled, isLoadingKb, onEnableKB]);
+
+  const knowledgeBaseDescription = useMemo(() => {
+    return isKnowledgeBaseEnabled ? (
+      <span data-test-subj="kb-installed">
+        {i18n.KNOWLEDGE_BASE_DESCRIPTION_INSTALLED(
+          enableKnowledgeBaseByDefault
+            ? KNOWLEDGE_BASE_INDEX_PATTERN
+            : KNOWLEDGE_BASE_INDEX_PATTERN_OLD
+        )}{' '}
+        {knowledgeBaseActionButton}
+      </span>
+    ) : (
+      <span data-test-subj="install-kb">
+        {i18n.KNOWLEDGE_BASE_DESCRIPTION} {knowledgeBaseActionButton}
+      </span>
+    );
+  }, [enableKnowledgeBaseByDefault, isKnowledgeBaseEnabled, knowledgeBaseActionButton]);
+
+  //////////////////////////////////////////////////////////////////////////////////////////
+  // ESQL Resource
+  const onEnableESQL = useCallback(
+    (enabled: boolean) => {
+      if (enabled) {
+        setupKB(ESQL_RESOURCE);
+      } else {
+        deleteKB(ESQL_RESOURCE);
+      }
+    },
+    [deleteKB, setupKB]
+  );
+
+  const esqlActionButton = useMemo(() => {
+    return isLoadingKb || !isESQLAvailable ? (
+      <></>
+    ) : (
+      <EuiButtonEmpty
+        color={isESQLEnabled ? 'danger' : 'primary'}
+        flush="left"
+        data-test-subj="esqlEnableButton"
+        onClick={() => onEnableESQL(!isESQLEnabled)}
+        size="xs"
+      >
+        {isESQLEnabled ? i18n.KNOWLEDGE_BASE_DELETE_BUTTON : i18n.KNOWLEDGE_BASE_INIT_BUTTON}
+      </EuiButtonEmpty>
+    );
+  }, [isLoadingKb, isESQLAvailable, isESQLEnabled, onEnableESQL]);
+
+  const esqlDescription = useMemo(() => {
+    return isESQLEnabled ? (
+      <span data-test-subj="esql-installed">
+        {i18n.ESQL_DESCRIPTION_INSTALLED} {esqlActionButton}
+      </span>
+    ) : (
+      <span data-test-subj="install-esql">
+        {i18n.ESQL_DESCRIPTION} {esqlActionButton}
+      </span>
+    );
+  }, [esqlActionButton, isESQLEnabled]);
+
+  return (
+    <>
+      <EuiTitle size={'s'}>
+        <h2>
+          {i18n.SETTINGS_TITLE}{' '}
+          <EuiBetaBadge iconType={'beaker'} label={i18n.SETTINGS_BADGE} size="s" color="hollow" />
+        </h2>
+      </EuiTitle>
+      <EuiSpacer size="xs" />
+      <EuiText size={'s'}>{i18n.SETTINGS_DESCRIPTION}</EuiText>
```

Comment:
As with `Anonymization`, I don't think we need the title here. Also, can we wrap this in an `<EuiPanel hasShadow={false} hasBorder paddingSize="l">` like the other tabs?

Knowledge Base:
<img width="1568" alt="image" src="https://github.com/elastic/kibana/assets/2946766/1bc12c89-287c-48dc-b1d3-4003435ee126">
Quick Prompts:
<img width="1572" alt="image" src="https://github.com/elastic/kibana/assets/2946766/1e07c58b-58ee-4edd-aaac-a2cb1bc21b2b">